src/server/plugins/modules.ts

```diff
@@ -6,9 +6,14 @@ import { Readable } from 'stream'
 import { init as initLexer, parse } from 'es-module-lexer'
 import MagicString from 'magic-string'
 import { cachedRead } from '../utils'
+import fs from 'fs'
+import { promisify } from 'util'
```

Comment:
We can do `import { promises as fs } from 'fs'` to get the Promise-based version directly.

Reply:
Resolved 👍